docs/src/components/Hero/SelectedLabel/SelectionBox/index.tsx

```diff
@@ -0,0 +1,62 @@
+import clsx from 'clsx';
+import React, { useEffect, useMemo, useState } from 'react';
+import styles from './styles.module.css';
+import Draggable from 'react-draggable';
+
+export enum DraggableId {
+  TOP_LEFT,
+  TOP_RIGHT,
+  BOTTOM_LEFT,
+  BOTTOM_RIGHT,
+  CENTER,
+}
+
+const SelectionBox: React.FC<{
+  propagationFunction: (
+    position: { x: number; y: number },
+    draggableIdentifier: DraggableId
+  ) => void;
+  draggableIdentifier: DraggableId;
+  children?: React.ReactNode;
+  isInteractive: Boolean;
+}> = ({
+  propagationFunction,
+  draggableIdentifier,
+  children,
+  isInteractive,
+}) => {
+  let classList = clsx(
+    styles.selectionBox,
+    draggableIdentifier == DraggableId.BOTTOM_LEFT ||
+      draggableIdentifier == DraggableId.BOTTOM_RIGHT
+      ? styles.boxLower
+      : styles.boxUpper,
+    draggableIdentifier == DraggableId.BOTTOM_LEFT ||
+      draggableIdentifier == DraggableId.TOP_LEFT
+      ? styles.boxLeft
+      : styles.boxRight
```

Comment:
Can we assign those if condition to a named variable?

draggableIdentifier == DraggableId.BOTTOM_LEFT ||
      draggableIdentifier == DraggableId.BOTTOM_RIGHT
      
draggableIdentifier == DraggableId.BOTTOM_LEFT ||
      draggableIdentifier == DraggableId.BOTTOM_RIGHT